src/app/loaders/express/index.ts

```diff
@@ -166,6 +141,16 @@ const loadExpressApp = async (connection: Connection) => {
   // New routes in preparation for API refactor.
   app.use('/api', ApiRouter)
 
+  // Serve static client files only in prod
+  if (config.nodeEnv === Environment.Prod) {
+    const frontendPath = path.resolve('dist/frontend')
+    app.use(express.static(frontendPath))
+
+    app.get('*', (_req, res) => {
```

Comment:
the order matters here, right? if we do `app.get('*')` before `app.use('/api`, that'll mess up our APIs as far as I can tell. would be good to add a comment here stating that it's important for this block to be below all our other `app.use`.

Reply:
Done